package.json

```diff
@@ -55,6 +55,7 @@
     "shared/**/*.{ts,tsx}": "npm run lint:shared"
   },
   "dependencies": {
+    "react-bootstrap-icons": "^1.7.2",
```

Comment:
1. It is frontend-only dependency. Move to the appropriate package.json file and lock lib version